@xen-orchestra/lite/src/App/TreeView.tsx

```diff
@@ -99,7 +99,7 @@ const TreeView = withState<State, Props, Effects, Computed, ParentState, ParentE
             id: pool.$id,
             label: (
               <span>
-                <Icon icon='cloud' /> {pool.name_label}
+                <Icon icon='warehouse' color='#0085FF' /> {pool.name_label}
```

Comment:
There is no way to don't hardcode the hexa code ?

Reply:
I'll fix that in another PR and modify the Icon component to support colors in the theme palette.